src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/DoNotCatchGeneralExceptionTypesTests.cs

```diff
@@ -400,9 +386,9 @@ public static void TestMethod()
         }
 
         [Fact]
-        public void Basic_NoDiagnostic_ThrowNewWrapped()
+        public async Task Basic_NoDiagnostic_ThrowNewWrapped()
         {
-            VerifyBasic(@"
+            await VerifyVB.VerifyAnalyzerAsync(@"
```

Comment:
I think we should have separate method names for helpers validating presence of diagnostics and missing diagnostics. Right now there is no way to quickly differentiate tests in the file that validate presence of diagnostics and those that validate missing diagnostics. Also it is very easy to introduce a bug where you forget to add an annotation - having a separate API like `VerifyMissingInAnalyzerAsync` seems more appropriate.

Reply:
This was weird to start with, but in the end it turned out to not matter much. You still have to look carefully at the code to determine what the expected outcome is, and when you take that closer look it's immediately apparent whether or not diagnostics are included.

Reply:
I'm also looking into some long-term solutions, but the API intentionally only includes a minimum set of APIs.

Reply:
Sounds reasonable, but I think there will be authors who prefer different names, so might not hurt having a separate API that just delegates to the existing one.

Reply:
The API is defined in the consuming project, not in the framework. Authors can choose to use the names recommended in the Microsoft.CodeAnalysis.Testing library (as this project currently does), or introduce their own patterns.